src/effects/VST3/VST3Effect.cpp

```diff
@@ -447,6 +347,7 @@ bool VST3Effect::RealtimeAddProcessor(
       auto wrapper = std::make_unique<VST3Wrapper>(*mModule, mEffectClassInfo.ID());
       if(!wrapper->Initialize(settings, sampleRate, Vst::kRealtime, mProcessingBlockSize))
          throw std::runtime_error { "VST3 realtime initialization failed" };
+      wrapper->ConsumeChanges(settings);
```

Comment:
This is the only place where the processors get changed settings information, and it only happens before playback.